conan/internal/api/detect_api.py

```diff
@@ -397,12 +397,12 @@ def detect_default_compiler():
         clang, clang_version, compiler_exe = _clang_compiler()  # prioritize clang
         if clang:
             return clang, clang_version, compiler_exe
-        return
```

Comment:
I'd keep this `return None, None, None` to simplify the flow too

Reply:
Agree

Reply:
Agree, can  you please do this change?

I have been doing a quick check, testing this seems possible with a mock of ``_clang_compiler``, but probably not worth the complexity (too much mock for testing one ``if``), so I am good without tests for this PR

Reply:
Done 😄